### PR TITLE
Rename IceSSL namespace to Ice::SSL

### PR DESCRIPTION
**Pull request by pepone**
This PR renames IceSSL namespace to Ice::SSL, It also fixes SSL and Schannel spelling

### CODE REVIEW

File: cpp/src/Ice/Instance.cpp [hunk 1]
```diff
@@ -3,7 +3,7 @@
 //
 
 #include "Instance.h"
-#include "../IceSSL/SSLEngine.h"
+#include "../Ice/SSL/SSLEngine.h"
```
Comment (bernardnormier) [inline cpp/src/Ice/Instance.cpp:6]: I don't think we should use this include syntax.

If it's a public header, use "Ice/SSL/SSLEngine.h". If it's a private (local) header, use "SSL/SSLEngine.h".

File: cpp/include/Ice/SSL/OpenSSL.h [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
+#ifndef ICE_SSL_OPENSSL_H
+#define ICE_SSL_OPENSSL_H
 
 #include "Certificate.h"
 
 #include <openssl/pem.h>
 #include <openssl/x509v3.h>
 
-namespace IceSSL::OpenSSL
+namespace Ice::SSL::OpenSSL
```
Comment (bernardnormier) [inline cpp/include/Ice/SSL/OpenSSL.h:13]: If we keep this public API, I would prefer a prefix (OpenSSLCertificate) over a sub-namespace, just like we did for the authentication options.

**Review by bernardnormier — APPROVED** [review]

File: cpp/src/Ice/Instance.cpp (hunk 1 above)
Comment (externl) [reply cpp/src/Ice/Instance.cpp:6]: Seems like this is likely a typo.

File: cpp/src/Ice/SSL/SchannelCertificateI.cpp [hunk 3]
```diff
@@ -34,10 +34,10 @@ namespace
     };
     using CertInfoHolderPtr = shared_ptr<CertInfoHolder>;
 
-    class SCHannelX509ExtensionI : public X509Extension
+    class ScHannelX509ExtensionI : public X509Extension
```
Comment (externl) [inline cpp/src/Ice/SSL/SchannelCertificateI.cpp:37]: Is this casing correct? Shouldn't these be `Schannel`?

**Review by externl — APPROVED** [review]

**Pull request merged**